keps/sig-storage/3107-csi-nodeexpandsecret/README.md

```diff
@@ -279,14 +382,17 @@ TBD
   provider?** no.
 
 - **Will enabling / using this feature result in increasing size or count of
-  the existing API objects?** no.
+  the existing API objects?**
+  yes, this adds a new field to the API so it changes the size.
 
 - **Will enabling / using this feature result in increasing time taken by any
   operations covered by [existing SLIs/SLOs]?** no.
 
 - **Will enabling / using this feature result in non-negligible increase of
   resource usage (CPU, RAM, disk, IO, ...) in any components?** no.
 
+- **Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?** no
+
 ### Troubleshooting
```

Comment:
There are answers missing for troubleshooting: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#troubleshooting

for beta we need to provide a starting point for debugging the feature.

Reply:
@deads2k I have mentioned a couple of steps as the primary debugging steps.